### ISSUE TITLE
openVPN Server Wizard: Uncaught ArgumentCountError: Too few arguments to function cert_create()

### ISSUE THREAD

**Issue by oceanBT** [opnsense/core#4943]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues and I am convinced that mine is new.

**Describe the bug**

When using the openvpn wizard in the web gui  (wizard.php?xml=openvpn ), to create an new openvpn server, this error occurs:

```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function cert_create(), 7 passed in /usr/local/etc/inc/plugins.inc.d/openvpn/wizard.inc on line 631 and exactly 8 expected in /usr/local/etc/inc/certs.inc:212 Stack trace: #0 /usr/local/etc/inc/plugins.inc.d/openvpn/wizard.inc(631): cert_create(Array, '6083298b0c171', '2048', '825', Array, 'sha256', 'server_cert') #1 /usr/local/www/wizard.php(155) : eval()'d code(1): step12_submitphpaction() #2 /usr/local/www/wizard.php(155): eval() #3 {main} thrown in /usr/local/etc/inc/certs.inc on line 212
```
It seems to be broken since this [commit](https://github.com/opnsense/core/commit/71d6d0adeb3d9b046114c12d0d2e2386caee3c36). So, maybe @kulikov-a can help (?). 

**To Reproduce**

1. start  "new openvpn server wizard" and go step by step through it 
 
**Expected behavior**

The openvpn server should be generated after the wizard, without errors.

**Environment**

Software version used and hardware type if relevant, e.g.:

OPNsense 21.1.5-amd64

thanks!

**Comment by AdSchellevis** [comment]
@oceanBT can you try https://github.com/opnsense/core/commit/ca67acc9d8b048dfc31c33453dfc0d08c4dccb36 ? install using the following command on the console:

```
opnsense-patch  ca67acc
```

**Comment by rk-zero** [comment]
> @oceanBT can you try [ca67acc](https://github.com/opnsense/core/commit/ca67acc9d8b048dfc31c33453dfc0d08c4dccb36) ? install using the following command on the console:
> 
> ```
> opnsense-patch  ca67acc
> ```

works, i could even install the patch and resend the form with F5. Thanks!

**Comment by AdSchellevis** [comment]
@rk-zero thanks for confirming

**Comment by gianks** [comment]
> > @oceanBT can you try [ca67acc](https://github.com/opnsense/core/commit/ca67acc9d8b048dfc31c33453dfc0d08c4dccb36) ? install using the following command on the console:
> > ```
> > opnsense-patch  ca67acc
> > ```
> 
> works, i could even install the patch and resend the form with F5. Thanks!

Confirmed, just had the same issue and using CLI and resending the form worked as expected.
Thanks a lot!

**Comment by fichtner** [comment]
It’s already fixed in 21.1.6.

**Comment by gianks** [comment]
> It’s already fixed in 21.1.6.

Indeed, updated today, did not see that!
